SignUp.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" type="text/css" href="resources/css/style.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,300&display=swap" rel="stylesheet">
+    <link href="resources/css/queries.css" type="text/css" rel="stylesheet">
+    <link href="vendors/css/Grid.css" rel="stylesheet" type="text/css">
+    <link href="vendors/css/normalize.css" rel="stylesheet" type="text/css">
+
+    
+    <script src="https://use.fontawesome.com/releases/v5.15.3/js/all.js" data-auto-replace-svg="nest"></script>
+
+    <title>Sabka Bazaar Groceries Pvt. Ltd. </title>
+</head>
+<body>
+    <header>
+ <div class="row">
+        <!--Top Navigation-->
+            <div class="col span-1-of-2">
+                &nbsp;
+            </div>
+            
+            <div class="col span-1-of-2">
+                <ul class="TopNav">
+              <li> <a href="Login.html">SignIn</a> </li>
+               <li> <a href="SignUp.html">Register</a></li>
+               </ul>
+           </div>
+        </div>
+    </div>
+       <!--Top navigation Ends-->
+
+        <!--Main Navigation-->
+
+        <nav class="navi">
+        <div class="row">  
+          <div class="col span-1-of-3">
+            <a href="index.html"> <img alt="Logo" src="resources/img/logo.png" class="logo-image"> </a> </li>
+          </div>
+
+          <div class="col span-1-of-3">
+           
+                <ul class="main-nav">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Products</a></li>   
+                </ul>
+         </div>
+          <div class="col span-1-of-3">
+              <div class="cart">
+            <a href="#" class="cart-link"><img src="resources/img/cart.svg" alt="cart" class="cart-img"> 0 items </a>
+         </div>
+         </div>
+        </nav>
+
+</header>
```

Comment:
dont copy header complete markup on every page. Create a template of header that you can add on any page